components/feature/awesomebar/src/main/java/mozilla/components/feature/awesomebar/AwesomeBarFeature.kt

```diff
@@ -26,15 +26,24 @@ import mozilla.components.feature.tabs.TabsUseCases
 class AwesomeBarFeature(
     private val awesomeBar: AwesomeBar,
     private val toolbar: Toolbar,
-    private val engineView: EngineView? = null
+    private val engineView: EngineView? = null,
+    private val onEditStart: (() -> Unit)? = null,
```

Comment:
nit: We could specify the default behaviour here e.g. ```= { hideAwesomeBar }```.

Reply:
That's what I tried first but it wasn't visible from there... and I was pretty sure I used something like this before. 🤔

Reply:
@pocmo I think if you define val inside the class declaration, not as a parameter you can do it
`private val onEditStart = onEditStart ?: ::hideAwesomeBar`